hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/OzoneManager.java

```diff
@@ -2882,17 +2882,18 @@ public KeyInfoWithVolumeContext getKeyInfo(final OmKeyArgs args,
         perfMetrics.getGetKeyInfoResolveBucketLatencyNs(),
         () -> resolveBucketLink(resolvedVolumeArgs));
 
-    if (isAclEnabled) {
-      captureLatencyNs(perfMetrics.getGetKeyInfoAclCheckLatencyNs(), () ->
-          checkAcls(ResourceType.KEY, StoreType.OZONE, ACLType.READ,
-              bucket.realVolume(), bucket.realBucket(), args.getKeyName())
-      );
-    }
```

Comment:
There is a similar call in `lookupKey` which is not moved into the try-catch block.

Reply:
+1

Reply:
nice catch